index.html

```diff
@@ -144,7 +144,7 @@ <h2 class="projectName">Facebook 360</h2>
             <li class="lang">css</li>
             <li class="lang">javaScript</li>
           </ul>
-          <button type="button">See Project</button>
+          <button class="pop-btn" id="2" type="button">See Project</button>
```

Comment:
- [ ] Please note that the should popup follows the design for both desktop and mobile version from the chosen template in Figma (e.g., font, colours, images, tests, margins).
-Your Design
![screenshot (6)](https://user-images.githubusercontent.com/44624138/141280994-5777d80e-59b5-47f9-bd87-e961a4f2972e.png)
- Figma Design
![screenshot (7)](https://user-images.githubusercontent.com/44624138/141282279-48b45331-194a-4b8c-a1a6-87ff90942626.png)